view/templates/settings/delegation.tpl

```diff
@@ -10,7 +10,7 @@
 {{if !$is_child_user}}
 	<h3>{{$l10n.account_header}}</h3>
 	<div id="add-account-desc" class="add-account-desc"><p>{{$l10n.account_desc}}</p></div>
-	<p><a href="register">{{$l10n.add_account}}</a></p>
+	<p><a class="btn btn-primary" href="register">{{$l10n.add_account}}</a></p>
```

Comment:
Please use a real `<button>` element. This is important for accessibility.

Reply:
So an <a> tag with a <button> inside it?

Reply:
Done.

Reply:
Done.